README.md

```diff
@@ -174,6 +174,13 @@ EOS
 
 The directory [docs](./docs/) provides descriptions of more advanced usages such as training or benchmarking.
 
+## Slack
+
+We have a Slack workspace for developers and users to ask questions and discuss a variety of topics.
+
+ * https://daac-tools.slack.com/
+ * Please get an invitation from [here](https://t.co/ikroMYhRsm).
```

Comment:
Short links should not be used.
This link depends on the third party service and is no guarantee that it is permanent.

Reply:
fixed, https://github.com/daac-tools/vibrato/pull/131/commits/7b67deebdc43cf79b3b06f1ce97ef39039de8f02